docs/tutorial.md

```diff
@@ -3,217 +3,904 @@ id: tutorial
 title: Tutorial
 ---
 
-## Learn Formik by Building it.
+## Before we start
 
-My talk at React Alicante is really the best way to get started. It introduces the library (by watching me build a mini version of it) and demos how to build a non-trivial form (with arrays, custom inputs, etc.) using the real thing. It's around 54 minutes long.
+Welcome to the Formik tutorial. This will teach you everything you need to know to build simple and complex forms in React.
 
-<iframe width="800" height="315" src="https://www.youtube.com/embed/oiNtnehlaTo" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen title="Taming Forms in React - Jared Palmer"></iframe>
+If you're impatient and just want to start hacking on your machine locally, checkout [the 60-second quickstart](@todo).
```

Comment:
Should the 60-second quickstart be hooked up? If it should be left as a todo, we should add "(todo)" after it so people aren't confused.

Reply:
After reading this today, I don't think this is really a tutorial, but more of a "thinking" in formik. It's a start, but lots more to do.